lib/vyasa_web/components/contexts/components.ex

```diff
@@ -390,28 +390,39 @@ defmodule VyasaWeb.Context.Components do
   end
 
   @doc """
-  TODO: implement a reddit-top-comment-like UI for this
   A brief view of a sheaf, showing the contextually relevant information about it.
+  Shows a barebones, brief summary of a sheaf, which gives just enough context for someone to
```

Comment:
probably should be sheaf body as title and first mark

Reply:
you're right, will get back to this when i'm doing ui iteration again after the current immediate next set of tasks